src/components/Dropdown/DropdownOption.st.css

```diff
@@ -31,20 +64,38 @@
 
 .root:not(:selectable) .title {
     -st-mixin: TPAText(
-        MainTextColor 'value(colorPlaceholder)'
+        MainTextColor value(colorPlaceholder)
+    );
+}
+
+.overrideStyleParams:not(:selectable) .title {
+    -st-mixin: TPAText(
+        MainTextColor value(CurrentItemTextDisabledColor),
+        MainTextFont value(CurrentTitleTextFont)
     );
 }
 
 .root:sectionTitle .title {
     padding: 0;
 }
 
-.subtitle {
-    -st-extends: TPAText;
+.overrideStyleParams .subtitle {
+     -st-mixin: TPAText(
+        MainTextColor value(CurrentTitleTextColor),
+        MainTextFont value(CurrentTitleTextFont)
+    );
+}
+
+.overrideStyleParams:not(:selectable) .subtitle {
     -st-mixin: TPAText(
-        MainTextColor "value(colorPlaceholder)",
-        MainTextFont "{theme: 'Body-M', size: '14px', lineHeight: '20px'}"
+        MainTextColor value(CurrentItemTextDisabledColor),
+        MainTextFont value(CurrentTitleTextFont)
     );
+}
+
+.subtitle {
+    font-size: 14px !important;
```

Comment:
Will this still work without the `!important`?

Reply:
does not work without the `!important` =\